File: generated-sources/php-ze-ph/mojang-sessions/src/App/DTO/PlayerSkinMetadata.php
<?php

namespace App\DTO;

use Articus\DataTransfer\Annotation as DTA;

/**
 * Contains details about the skin
 */
class PlayerSkinMetadata
{
    /**
     * The player model, currently only \&quot;slim\&quot; (Alex) is valid, for Steve&#39;s model this property must be absent.
     * @DTA\Data(field="model", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"string"})
     * @var string
     */
    public $model;
}

File: generated-sources/kotlin-spring/mojang-sessions/src/main/kotlin/org/openapitools/model/PlayerSkinMetadata.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import com.fasterxml.jackson.annotation.JsonValue
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * Contains details about the skin
 * @param model The player model, currently only \"slim\" (Alex) is valid, for Steve's model this property must be absent.
 */
data class PlayerSkinMetadata (

        @JsonProperty("model") val model: PlayerSkinMetadata.Model? = null
) {

    /**
    * The player model, currently only \"slim\" (Alex) is valid, for Steve's model this property must be absent.
    * Values: slim
    */
    enum class Model(val value: String) {
    
        @JsonProperty("slim") slim("slim");
    
    }

}

